fhd_utils/l_m_n.pro

```diff
@@ -0,0 +1,26 @@
+pro l_m_n, obs, psf, obsdec=obsdec, obsra=obsra, dec_arr=dec_arr, ra_arr=ra_arr,$
+    l_mode=l_mode,m_mode=m_mode,n_tracked=n_tracked
```

Comment:
Drop `n_tracked` if you make this a function

Reply:
done